frontend/tests/ui/util.py

```diff
@@ -432,44 +431,17 @@ def create_problem(driver, problem_alias, has_privileges=True):
         assert (('/problem/%s/edit/' % problem_alias) in
                 driver.browser.current_url), driver.browser.current_url
     else:
-        assert_get_alert(driver, contents_element)
+        contents_element.submit()
+        assert_alert_is_shown(driver)
 
 
-def assert_get_alert(driver, submit_element):
-    ''' When an identity tries to create something it should get a message. '''
+def assert_alert_is_shown(driver):
+    '''When an identity tries to perform an action without permission,
```

Comment:
hmmm no me había dado cuenta que no cabía

```suggestion
    '''An alert is shown when an action is attempted without permission.'''
```

Reply:
Listo